pandas/compat/__init__.py

```diff
@@ -138,16 +138,6 @@ def raise_with_traceback(exc, traceback=Ellipsis):
 raise_with_traceback.__doc__ = """Raise exception with existing traceback.
 If traceback is not passed, uses sys.exc_info() to get traceback."""
 
-
-# dateutil minimum version
-import dateutil
-
```

Comment:
we prob need to add a check on setup.py i think though

Reply:
dateutil is a required dependency though. Not sure why we would need a check.

Reply:
and this is where its guaranteed no? though I guess its the same as with numpy. you can install an older version but that would break in weird ways, so ok .